include/OperatingSystemHelpers.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * OperatingSystemHelpers.h - Helpers for OS related concerns
+ *
+ * Copyright (c) 2025- Michael Gregorius
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LMMS_OPERATINGSYSTEMHELPERS_H
+#define LMMS_OPERATINGSYSTEMHELPERS_H
+
+#include "lmmsconfig.h"
+
+#include "qnamespace.h"
+
+
+namespace lmms
+{
+
+constexpr char LADSPA_PATH_SEPERATOR =
+#ifdef LMMS_BUILD_WIN32
+';';
+#else
+':';
+#endif
+
+// Abstract away GUI CTRL key (linux/windows) vs ⌘ (apple)
+constexpr const char* UI_CTRL_KEY =
+#ifdef LMMS_BUILD_APPLE
+"⌘";
+#else
+"Ctrl";
+#endif
+
+constexpr const char* UI_ALT_KEY =
+#ifdef LMMS_BUILD_APPLE
+"Option";
+#else
+"Alt";
+#endif
+
+constexpr const char* getOSSppecificModifierKeyString()
+{
+#ifdef LMMS_BUILD_APPLE
+    return UI_ALT_KEY;
+#else
+    return UI_CTRL_KEY;
+#endif
+}
+
+constexpr Qt::KeyboardModifier getOSSpecificModifierKey()
```

Comment:
Curious, why is this and `getOSSppecificModifierKeyString` are functions?

Reply:
Might be to simplify the code and reduce the usage of macros to this one file, which is a good way of reducing macros, and some verbosity here.

a better name for this function shall be `getCopyModifier()` similar to the naming in my previous comment.

Reply:
> Curious, why are this and `getOSSppecificModifierKeyString` functions?

I think it was mainly because all the `ifdef`s were previously scattered in the "active" code and therefore my first instinct was to move it to a single place but still with a "functiony" mindset. Using just `constexpr` variables as done by your further changes is the better and less complex solution though.